_posts/2023-10-21-beneath-the-shell.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: Beneath the shell
+author: Tommaso Becca
+---
+
+From the Watcher archives, decrypted and translated from records downloaded on Lateria'sei. The file describes a brief history of the Ka'het, as written by a Builder engineer three months before the first wave of insurrections.
+
+<img class="centered" src="/images/blog/ka-het/header.jpg" width="840" height="350" />
+
+## Early concepts: the "super-sizing"
+
+The story begins over two centuries before the first Ka'het saw the light; not when the proposal was first conceived, but when it was first realized, through a tech demo that applied what is known as "super-sizing" an organism. This super-sizing process led to the creation of a three meter long, half a meter wide worm, incapable of sustaining its own weight in a gravitational field or surviving on its own. Though its appearance would be much different from what followed, and it had no functions beyond eating and growing, nothing would have been possible without the breakthroughs of the super-sizing project - as a matter of fact, it wasn't long after this that the Fetri'sei program began to make its first steps.
+
+<img class="centered" src="/images/blog/ka-het/super-sized.jpg" width="400" height="300" />
+
+## The Fetri'sei, and the exploration of deep space
+
+It was around this time that, between the various departments of the interstellar authority, the 'sei group was nearing their peak: managing over two dozen stations in the core systems (with three more in various stages of construction) and a number of smaller projects on the capital's surface, they had grown far beyond their initial role of space platform operators and made many inside the government uneasy about the power they could exert. To break the hegemony that was starting to form, the 'mar group was created, intended to take some of the most promising space-based projects and develop them to completion - for instance, the then-new program of automating a large part of the interstellar traffic through robotic spacecrafts and surface bases. The 'sei group's response didn't have to wait long: a competing program, which instead of automated spacecrafts put super-sized organisms at the helm and would have been capable of space travel, exploration and self replication; thus, the Fetri were born.
```

Comment:
That's a run-on sentence no matter how it's written. Also, they _were_ capable of space travel, exploration, and self replication, right?

```suggestion
It was around this time that, between the various departments of the interstellar authority, the 'sei group was nearing their peak: managing over two dozen stations in the core systems (with three more in various stages of construction) and a number of smaller projects on the capital's surface, they had grown far beyond their initial role of space platform operators and made many inside the government uneasy about the power they could exert. To break the hegemony that was starting to form, the 'mar group was created, intended to take some of the most promising space-based projects and develop them to completion - for instance, the then-new program of automating a large part of the interstellar traffic through robotic spacecrafts and surface bases. The 'sei group responded with a competing program, which, instead of automated spacecrafts, put super-sized organisms at the helm. These organisms were capable of space travel, exploration and self replication; thus, the Fetri were born.
```